test/unit/edgeware/enricher.spec.ts

```diff
@@ -774,7 +785,23 @@ describe('Edgeware Event Enricher Filter Tests', () => {
       }
     });
   });
-
+  /** offences events */
+  it('should enrich new offence event', async () => {
+    const kind = EventKind.Offence;
+    const event = constructEvent([ 'offline', '10000', true, offenceDetails ],
+      'offences', [ 'Kind', 'OpaqueTimeSlot', 'boolean', 'Option<OffenceDetails>[]' ]);
+    const result = await Enrich(api, blockNumber, kind, event);
+    assert.deepEqual(result, {
+      blockNumber,
+      data: {
+        kind,
+        offenceKind: 'offline',
+        opaqueTimeSlot: '10000',
+        applied: true,
```

Comment:
You should use the `constructBool()` helper call to generate the test value for `applied`.